static/docs/commands-reference/pipeline.md

```diff
@@ -17,7 +17,18 @@ positional arguments:
 
 ## Description
 
-Manage pipeline.
+A data pipeline, in general, is a chain of commands that process data files. It
+produces intermediate data and a final result. Machine Learning (ML) pipelines
```

Comment:
`For example, Machine Learning (ML) pipelines ...`

Reply:
Updated